src/PowerPing/Ping.cs

```diff
@@ -491,6 +510,12 @@ private PingResults SendICMP(PingAttributes attrs)
             return Results;
         }
 
+        private static ushort GenerateSessionId()
```

Comment:
Makes sense to move this function to Helper.cs, just because thats where most helpful functions live.

Reply:
Done!